sysutils/pfSense-pkg-System_Patches/files/usr/local/pkg/patches.inc

```diff
@@ -200,36 +195,35 @@ function bootup_apply_patches() {
 function patch_add_shellcmd() {
 	global $config;
 	$a_earlyshellcmd = &$config['system']['earlyshellcmd'];
```

Comment:
It must check if isset() or is_array() on $config['system']['earlyshellcmd'] before set the reference here. If it's not an array then it must be initialized as
```
if (!isset($config['system']['earlyshellcmd'])) {
    $config['system']['earlyshellcmd'] = array();
}
$a_earlyshellcmd = &$config['system']['earlyshellcmd'];
```
After that , the if(is_array()) from line 199 can go away